frontend/www/js/omegaup/components/arena/Contest.vue

```diff
@@ -177,21 +205,26 @@ export default class ArenaContest extends Vue {
     return this.activeProblem?.problem.alias ?? null;
   }
 
+  get deadline(): Date | boolean {
```

Comment:
```suggestion
  get deadline(): Date | null {
```

hay que intentar no mezclar objetos y primitivos. para eso está `null`. si se require otra cosa, nuestras amigas las uniones discriminadas son la opción.

también, sería mejor separar obtener el deadline del concepto "ya se terminó el concurso?", porque un deadline nulo medio implica que el concurso nunca termina.

Reply:
Listo